docs/source/tutorial/glance.rst

```diff
@@ -0,0 +1,359 @@
+Chainer at a Glance
+-----------------------
+
+.. currentmodule:: chainer
+
+Welcome to Chainer!
+
+Chainer is a rapidly growing neural network platform. The strengths of Chainer are:
+
+* 100% pure Python -- Chainer is developed from the beginning in Python, source code and errors follow the Pythonic way
+* Define by Run -- neural networks definitions are defined on-the-fly at run time, allowing for easier customisation
+* Broad and deep support -- Chainer is actively used for most of the current approaches for neural nets (CNN, RNN, RL, etc.), aggressively adds new approaches as they're developed, and provides support for many kinds of hardware as well as parallelization for multiple GPUs
+
+
+Mushrooms -- tasty or deathly?
+~~~~~~~~~~~~
+
+Let's take a look at a basic program of Chainer to see how it works. For a dataset, we'll work with the edible vs. poisonous mushroom dataset, which has over 8,000 examples of mushrooms, labelled by 22 categories including odor, cap color, habitat, etc.
+
+How will Chainer learn which mushrooms are edible and which mushrooms will kill you? Let's see!
+
+Full Code
+~~~~~~~~~~~~
+
+Here's the whole picture of the code:
+
+.. code-block:: python
+    
+    #!/usr/bin/env python
+    
+    from __future__ import print_function
+    
+    try:
+        import matplotlib
+        matplotlib.use('Agg')
+    except ImportError:
+        pass
+    
+    import chainer
+    import chainer.functions as F
+    import chainer.links as L
+    from chainer import training, datasets
+    from chainer.training import extensions
+    
+    import numpy as np
+    import sklearn.preprocessing as sp
+    
+    data_array = np.genfromtxt('mushrooms.csv', delimiter=',',dtype=str, skip_header=1)
+    labelEncoder = sp.LabelEncoder()
+    for col in range(data_array.shape[1]):
+        data_array[:, col] = labelEncoder.fit_transform(data_array[:, col])
+    
+    X = data_array[:, 1:].astype(np.float32)
+    Y = np.ndarray.flatten(data_array[:, 0].astype(np.int32))
+    train, test = datasets.split_dataset_random(datasets.TupleDataset(X, Y), int(data_array.shape[0] * .7))
+    
+    gpu_id = -1
+    
+    # Network definition
+    class MLP(chainer.Chain):
+    
+        def __init__(self, n_units, n_out):
+    	super(MLP, self).__init__()
+    	with self.init_scope():
+    	    # the size of the inputs to each layer will be inferred
+    	    self.l1 = L.Linear(n_units)  # n_in -> n_units
+    	    self.l2 = L.Linear(n_units)  # n_units -> n_units
+    	    self.l3 = L.Linear(n_out)  # n_units -> n_out
+    
+        def __call__(self, x):
+    	h1 = F.relu(self.l1(x))
+    	h2 = F.relu(self.l2(h1))
+    	return self.l3(h2)
+    
+    
+    model = L.Classifier(MLP(44, 2))
```

Comment:
Two-class classification should be done with `F.sigmoid_cross_entropy` for efficiency. So this line should be:
```
model = L.Classifier(
    MLP(44, 1), lossfun=F.sigmoid_cross_entropy, accfun=F.binary_accuracy)
```